Eatery Blue/UI/EateryCard/EateryMediumCardContentView.swift

```diff
@@ -157,9 +159,14 @@ class EateryMediumCardContentView: UIView {
         
         favoriteButton.snp.makeConstraints { make in
             make.leading.equalTo(subtitleLabel.snp.trailing).offset(4)
-            make.trailing.equalTo(imageView.snp.trailing).inset(12)
-            make.width.height.equalTo(20)
-            make.top.equalTo(imageView.snp.bottom).offset(12)
+            make.trailing.equalTo(imageView.snp.trailing)
+            make.size.equalTo(44)
+            make.top.equalTo(imageView.snp.bottom)
+        }
```

Comment:
As mentioned above, test this with different screen sizes.

Reply:
looks good on iphone se 3rd generation, iphone 15 pro, iphone 15 pro max, and ipad pro 4th generation 11 inch